apps/evi/agent/tools/turbo.ts

```
@@ -32,16 +33,16 @@ export default defineDynamic({
               return { success: false as const, error: `No Vercel OIDC token available: ${error instanceof Error ? error.message : String(error)}` }
             }
             let token: string
-        try {
-          token = await exchangeTurboToken(oidc, teamSlug)
-        }
-        catch {
-          return { success: false as const, error: 'Turborepo token exchange failed; remote caching is unavailable for this run.' }
-        }
+            try {
+              token = await exchangeTurboToken(oidc, teamSlug)
+            }
```

ESLint

[error] Closing curly brace does not appear on the same line as the subsequent block.

(brace-style)